medcat/utils/ner/deid.py

```diff
@@ -62,18 +62,73 @@ def train(self, json_path: Union[str, list, None],
               *args, **kwargs) -> Tuple[Any, Any, Any]:
         return super().train(json_path, *args, train_nr=0, **kwargs)  # type: ignore
 
-    def deid_text(self, text: str, redact: bool = False) -> str:
+    def get_chunks(self, text:str, de_id_pipe,maximum_tkns) -> List[List[Any]]:
+
+        blocks = [[]]
+        if 'Roberta' in de_id_pipe.ner_pipe.tokenizer.__class__.__name__:
```

Comment:
I think this is a good check, but I would extract the code below to a separate method, the name of which is explicit about it being for roberta models (i.e `get_chunks_roberta`) and a docstring that specifies that it's designed for these and is unlikely to work with anything else.